h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -138,6 +128,23 @@ public int compare(Integer o1, Integer o2) {
       return this;
     }
 
+    public static void calculateVarimpMultinomial(double[] magnitudes, Integer[] indices, double[][] betaNorm) {
+      for (int i = 0; i < betaNorm.length; ++i) {
+        for (int j = 0; j < betaNorm[i].length; ++j) {
+          double d = betaNorm[i][j];
+          magnitudes[j] += d < 0 ? -d : d;
```

Comment:
Is this correct? Should we instead consider the aggregated coefficient (= move this to the outer loop)?

I don't know - asking because it is not clear to me.

Reply:
Here the first for loop is over all classes.

The second for loop  is over all coefficients for a fixed class.  

Line 135 is basically adding the magnitudes of say coefficient j of all classes together.  It is aggregating coefficients over all classes and should be correct.  

Say, we have 3 classes and 5 coefficients.  Then, for class 0, it will go over all 5 coefficients and add the magnitude of the 5 coefficients for class 0.  Next, it will do the same for class 1, add the 5 coefficients into magnitude... and so on.  Does this make sense?

Reply:
I see, so I misunderstood the code - it is actually doing what I thought it should be doing based on your description. Thank you @wendycwong 

Could you add a (java) unit test for this method - it should be relatively easy to do (since this is a static method).

Reply:
Sure.